readme.md

```diff
@@ -101,3 +101,176 @@ In the console...
+
+Today, you are going to use Sinatra to listen for HTTP requests and serve HTML (rendered from ERB templates) in response.
+
+### Part 3.1: Root Route, Home View, and Layout View
+
+1. Run your application: `ruby app.rb` and go to localhost:4567 in your browser... Sinatra doesn't know this ditty.
+
+  - First just define the route in the General Routes section of app.rb as described by Sinatra:
+    ```ruby
+    get '/' do
+      "Hello World"
+    end
+    ```
+  Now head back to localhost:4567 and check it out!
+
+  - So maybe not super exciting but moving in the right direction
+
+  - Replace `"Hello World"` with HTML rendered from an ERB template, home. Our General Routes section of app.rb now reads:
+    ```ruby
+    get '/' do
+      erb :home
+    end
+    ```
+
+- [erb](http://www.sinatrarb.com/intro.html#Views%20/%20Templates) is a method provided by Sinatra which uses Ruby's [ERB (Embedded RuBy)](https://en.wikipedia.org/wiki/ERuby) templating system to take a template string (text with ruby written in it) and produce an output string after evaluating the embedded ruby.
+
+- If we tried to go to localhost:4567 now, we'd get an error saying __No such file or directory @ rb_sysopen - /path/to/tunr_sinatra/views/home.erb__
+
+    - When we give the `erb` method an argument of `:home`, the method looks in the __views/__ directory of the project for a file named home.erb and takes the contents of that file as its template string.
+
+    - Make that __views/__ directory and add a file home.erb.
+
+    - In home.erb, first add some HTML (no need to worry about embedding ruby just yet) and check it out at localhost:4567
+
+- The erb method will also check for a layout.erb file where you define the part of the template shared by all the pages in our app (e.g. head/body elements, header/footer, navigation bar, etc). In layout.erb need to include a line of embedded ruby with the keyword `<%= yield %>`. This tells the method to render the named template (`:home`) at this point in the layout.erb template. In the below steps, regularly check out localhost:4567 and consider the effects of your code changes
+
+    - Make a layout.erb file in views/
+
+    - As always, start simple, maybe just a heading. Check it out in the browser
+
+    - Then expand adding head and body elements, layout.erb should produce well formed HTML and since there is no embedded ruby in it yet, it should be well formed HTML
+
+    - Create a __public/__ directory in the root of your project. By default, Sinatra looks for static assets in a directory named __public/__ in the root directory of the project. In the __public/__ create a stylesheet.
+
```

Comment:
maybe hint here about how to link to files in the public folder, i.e. dont include the word `public`